### DISCUSSION TITLE
Modelling BS 8414 Testing

### DISCUSSION THREAD

**Discussion by Moabismywashpot** [firemodels/fds#12516] [Q&A]
How would someone go about modelling BS 8414 testing in FDS? I am new to FDS, and feel like I do not have enough technical knowledge to fully understand what I am trying to do, however have chosen to do this for my dissertation so am in need of guidance.

**Comment by drjfloyd** [comment]
We cannot do your disseration work for you. Learning to use FDS is not something that happens overnight. It will take some time.   

Start by reading all the FDS docmentation: Technical Reference Guide, User's Guide, Verification Guide, and Validation Guide. All the input files for examples in the User's Guide and Verification Guide are provided as part of the FDS installation. Validation Guide inputs can be found in the github repository under the Validation folder. Look for example cases or Validation cases that have commonalities with what you are trying to model.  Start with simple problems. Don't try to jump immediately into to modeling your full case. Think about how you can break your final case up into simple model problems focusing on some aspect of the final case and work on those simpler problems to ensure you understand how to model them before combining into the final more complex case.

**Reply by Moabismywashpot** [reply]
Thank you for the response. I found it very helpful to break the model down and have been looking at the guides provided. I was wondering if you would be able to help me understand a problem I am having. At the moment I am trying to model a wooden crib as a simple model with a prescribed fuel (using propane to begin with) and HRRPUA rate taken from the guidance in BS 8414. However whilst the model appears to work when the time is limited, when trying to scale the model up to the full time (30 minutes) the block appears to burn only for about 2 minutes before it appears to extinguish. At the moment I haven't given the crib material properties, could this be the problem? Any help would be greatly appreciated

**Reply by rmcdermo** [reply]
Is the domain sealed?  Do you have enough air?  Sounds like you are running out of oxygen to burn.

**Reply by Moabismywashpot** [reply]
Yes, I did not set any boundary conditions for the mesh, I will rerun it and see if that works, thanks!

**Reply by Moabismywashpot** [reply]
I am now looking to model the PIR insulation using thermal properties of the material, an ignition temperature and a fixed mass loss rate. It is possible to prescribe species release rate to directly to a material so that I can prescribe the CO yield, HCN yield and Soot Yield? Or does this have to be via a REAC line? Thanks!

**Comment by mcgratta** [comment]
If you declare CO, HCN, and SOOT to be SPECies, like
```
&SPEC ID='CARBON MONOXIDE' /
```
Then you can specify the mass loss rates of all these species on the SURF line. Your REAC line would identify which of these species is to combust.

**Reply by Moabismywashpot** [reply]
In the end I looked at using the formula of the fuel, and prescribed the amounts of species in in the REAC line as such:

&SPEC ID = 'Polyisocyanurate', FORMULA = CH1O0.19N0.11 /
&REAC FUEL = 'Polyisocyanurate', HCN_YIELD = 0.01, CO_YIELD = 0.038, SOOT_YIELD =0.1, HEAT_OF_COMBUSTION = 26300. /

Would this produce the same outcome?

**Comment by mcgratta** [comment]
I am not sure what you mean by the "same outcome". In any case, you can test your inputs by creating a simple test case where you have the fire in a closed room that burns for, say, 10 s, and then is turned off. You can specify `MASS_FILE=T` on the `DUMP` line, and then look at the total amount of species produced. See if they agree with estimates you can obtain with pencil and paper.
